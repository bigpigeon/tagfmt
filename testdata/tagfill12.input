//tagfmt -f "json=upper(:tag)"

package main

type OrderDetail struct {
    ID       string   `json:"Id"`
    UserName string   `json:"User"`
    OrderID  string   `json:"OrderID"`
    Callback string   `json:"Callback"`
    Address  []string `json:"Address"`
}
