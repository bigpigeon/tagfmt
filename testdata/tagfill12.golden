//tagfmt -f "json=upper(:tag)"

package main

type OrderDetail struct {
	ID       string   `json:"ID"`
	UserName string   `json:"USER"`
	OrderID  string   `json:"ORDERID"`
	Callback string   `json:"CALLBACK"`
	Address  []string `json:"ADDRESS"`
}
